products/firefox_android/localize_android.md

```diff
@@ -0,0 +1,45 @@
+# Firefox for Android - How to localize
+
+Please make sure to go through the [Firefox for Android README](/products/firefox_android/README.md) first to get a clear overview of what this project covers.
```

Comment:
Actually, re-reading it, it wasn't really clear 👎(it was in my head). Feel free to drop this paragraph.